src/Servers/IIS/AspNetCoreModuleV2/CommonLib/debugutil.cpp

```diff
@@ -22,11 +22,36 @@ inline HMODULE g_hModule;
 inline SRWLOCK g_logFileLock;
 inline HANDLE g_stdOutHandle = INVALID_HANDLE_VALUE;
 
+std::wstring GetDateTime()
+{
+    std::chrono::milliseconds milliseconds =
+        std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch());
+    std::chrono::seconds seconds = std::chrono::duration_cast<std::chrono::seconds>(milliseconds);
+    milliseconds = std::chrono::duration_cast<std::chrono::milliseconds>(milliseconds - seconds);
+
+    time_t t = seconds.count();
+    tm time;
+    // convert time to utc
+    gmtime_s(&time, &t);
+
+    constexpr auto size = sizeof("2019-11-23T13:23:02.000Z");
+    wchar_t timeString[size];
+
+    // format string to ISO8601 with additional space for 3 digits of millisecond precision
+    std::wcsftime(timeString, size, L"%FT%T.000Z", &time);
+
+    // add millisecond part
+    // 5 = 3 digits of millisecond precision + 'Z' + null character ending
+    swprintf(timeString + size - 5, 5, L"%03dZ", (int)milliseconds.count());
+
+    return std::wstring(timeString);
+}
+
 HRESULT
 PrintDebugHeader()
 {
     // Major, minor are stored in dwFileVersionMS field and patch, build in dwFileVersionLS field as pair of 32 bit numbers
-    LOG_INFOF(L"Initializing logs for '%ls'. %ls. %ls.",
```

Comment:
Removing the `.` because `GetProcessIdString()` already returns the `.` and my log OCD wasn't happy with `"Process ID: 1029.."`